services/QuillLMS/spec/models/classroom_unit_activity_state_spec.rb

```diff
@@ -77,12 +77,12 @@
 
     it 'caches data about the assignment' do
       lesson_data = { 'classroom_unit_id' => cua.classroom_unit.id,
```

Comment:
If you move the first key-value down a line, this will get rid of the added whitespace:

```
lesson_data = { 
  'classroom_unit_id' => cua.classroom_unit.id,
  'classroom_unit_activity_state_id' => cua.id,
  'activity_id' => cua.unit_activity.activity.id,
  'activity_name' => cua.unit_activity.activity.name,
  'unit_id' => cua.classroom_unit.unit_id,
  'completed' => cua.completed,
  'visible' => cua.unit_activity.visible 
}
```